docs/tutorials/new-feature-tutorial.md

```diff
@@ -145,10 +151,22 @@ information on writing and running tests, see the
 
 ### Documentation changes
 
-After implementing the new feature, you should
-document it and update any existing documentation that might be
-relevant to the new feature. For more information on the kinds of
-documentation Zulip has, see [Documentation](../documentation/overview.md).
+After implementing the new feature, you should document it and update
+any existing documentation that might be relevant to the new feature.
+For detailed information on the kinds of documentation Zulip has, see
+[Documentation](../documentation/overview.md).
+
+**General user documentation:** You will likely need to at least create
+or update a `/help` article for general users about your new feature.
+See [User documentation](../documentation/user.md) for more detailed
+information about writing and editing feature `/help` articles.
```

Comment:
I think s/general users/end users/ would be more typical terminology?   Also, I wonder if it's worth working in that often you'll just want to extend existing documentation for existing features that might be modified by your new setting, rather than making a new page, and also link to your new documentation from appropriate related articles.

Reply:
There was one sub-heading in the existing docs with ['General user documentation'](https://zulip.readthedocs.io/en/latest/documentation/overview.html#general-user-documentation). I've updated all the references in this tutorial and that sub-heading to be 'end user'.

Also, took out 'create and update' here and replaced with 'update, extend and link to'. That way the focus is engaging with the existing documentation. 

In the 'User documentation' tutorial, there's a [list of tips for adding a new article](https://zulip.readthedocs.io/en/latest/documentation/user.html#writing-documentation). Maybe another follow-up here would be adding a list of tips for updating existing documentation beforehand? I imagine what exists now was written when there was less end user documentation in general. Now that there is a good base, a slight shift of emphasis could be worth doing.

Reply:
These are all good ideas.

Can you start a thread about this in #documentation on chat.zulip.org?  I think it's worth having a bit of a broader conversation about these topics.  E.g. maybe we want to just lean into to calling /help/ the "help center" to make things clearer.